tools/import/index.html

```diff
@@ -26,8 +26,11 @@ <h1>Bulk Import</h1>
                     <label for="apiKey-input">API Key</label>
                     <input id="apiKey-input" type="password" autocomplete="apiKey" required/>
                 </div>
-                <div class="field">
-                    <label for="url-input">URLs</label>
+                <div class="field" id="url-drop-zone">
+                    <div style="display: flex; justify-content: space-between;">
+                        <label for="url-input">URLs</label>
+                        <span style="{align-content: flex-end; font-size: 6px}"><em>Drop text file to populate.</em></span>
```

Comment:
Help text should go below the field.

https://spectrum.adobe.com/page/help-text/

You should be able to use the `.field-help-text` text to style the content correctly as well.